### DISCUSSION TITLE
Game stutter/slight freeze at the end of a map with storyboard on

### DISCUSSION THREAD

**Discussion by bytenekos** [ppy/osu#30057] [Q&A]
Hi! I'm not sure if I should be making an issue for this or not so making this discussion

When playing a map and you reach the end, on some maps there's a stutter that happens if you have storyboards on (see video)

https://github.com/user-attachments/assets/8931f45d-cd9e-48d8-8c7f-e8259823a97d

I'm assuming it's got to do with the storyboard loading back up (idk exactly how storyboards work when you have background dim set to 0% and when there's a break making it show the storyboard again)
Stutter doesn't happen if I have background dim set below 100% (storyboard actually plays and stuff)

Map shown: https://osu.ppy.sh/beatmapsets/479671#osu/1024215

**Comment by peppy** [comment]
Probably definitely GC or disposal. Hopefully the latter because the former is a dead end.
